pkg/segment/results/mresults/tests/metricsresults_test.go

```diff
@@ -404,3 +406,167 @@ func TestCalculateInterval(t *testing.T) {
 		}
 	}
 }
+
+func Test_GetResults_GTR(t *testing.T) {
+	test_GetResults_Ops(t,
+		map[uint32]float64{
+			0:     45,
+			10800: 1045,
+		},
+		map[uint32]float64{},
+		[]structs.QueryArithmetic{
+			{
+				LHS:        1,
+				ConstantOp: true,
+				Operation:  utils.LetGreaterThan,
+				Constant:   1045,
+			},
+		})
+}
```

Comment:
Add some more test cases and/or points to test boundary conditions. For example, test with values of 45, 1045, and 1046

Similar comment for tests for the other operators